types/chromecast-caf-receiver/index.d.ts

```diff
@@ -78,4 +78,5 @@ declare global {
     type PlayerDataChangedEventHandler = (event: PlayerDataChangedEvent) => void;
     type RequestHandler = (request: framework.NetworkRequestInfo) => void;
     type BinaryHandler = (data: Uint8Array) => Uint8Array;
+    type TracksChangedHandler = () => void;
```

Comment:
No need for this as you are not adding an overload for addEventListener to support it.

In any case, the handler would receive an event.